app/main/views/suppliers.py

```diff
@@ -507,7 +508,7 @@ def become_a_supplier():
     try:
         frameworks = sorted(
             data_api_client.find_frameworks().get('frameworks'),
-            key=lambda framework: framework['slug'],
+            key=operator.itemgetter('framework', 'id'),
```

Comment:
I'm surprised this works, why is the first argument quoted?

Reply:
Actually, take that back, the quoting would make sense, except I don't see why the first argument is there -at all-. Probably me being daft tho at this point :)

Reply:
So it's so the list of frameworks being sorted are first sorted by framework, and then by id within that first sorting. Keeps the family members grouped.

Reply:
But yeah, I see what you mean now. Is keeping them grouped actually necessary. Probably not.

Reply:
Oh, framework-framework - right yep. So this is a change, somehow I assumed it was just intended to be the same ordering but written differently. If there's any reason to order at all then go for it, don't mind me :)

Reply:
@galund Have updated to drop to just sorting by ID